app/assets/javascripts/discourse/app/components/modal/login/footer.hbs

```diff
@@ -4,22 +4,22 @@
       <DButton
         @action={{@login}}
         id="login-button"
-        @form="login-form"
+        form="login-form"
         @icon="unlock"
         @label={{@loginButtonLabel}}
         @disabled={{@loginDisabled}}
         class="btn btn-large btn-primary"
-        @tabindex="2"
+        tabindex={{unless @showSecondFactor "2"}}
       />
     {{/unless}}
 
-    {{#if @showSignupLink}}
+    {{#if (and @showSignupLink (not @showSecondFactor))}}
```

Comment:
This feels a little odd. Maybe `showSignupLink` should return false if we're in a 2FA context?